Add all organization report task page

dekkers opened the pull request:
### Changes

- The all organization tasks page for reports was missing, this PR implements this. 
- Inconsistencies between the report task page and boefje and normalizer task pages are removed.
- Performance improvements when the user can access all organizations:
  - Do not provide an organization filter to the task endpoint
  - Use the stats endpoint for all organizations instead of sending a requests for each organization.
- Fixes a bug that the statistics were not filtered on task type anymore.

### QA notes

Create multiple organizations with report tasks and check the all organization task pages.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/contributor/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/contributor/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/contributor/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/contributor/templates/pull_request_template_review_qa.md) into your comment.

ammar92 reviewed (dismissed):
Looks good to me.

Just a note: The `_get` method seems quite redundant to me, but that can be refactored later on

dekkers commented:
I have added the organization column to the all organization reports page.

I have also changed the number of tasks on the all organization page to 150 just like the organization task pages have.

stephanie0x00 commented:
Yay looks good!
